components/input/__docs__/storybook-stories.jsx

```diff
@@ -231,4 +232,29 @@ storiesOf(FORMS_INPUT, module)
 				placeholder="My placeholder"
 			/>
 		</section>
+	))
+	.add('Inline Help', () => (
+		<section>
+			<h1 className="slds-text-title_caps slds-p-vertical--medium">
+				Inline Help
+			</h1>
+			<Input label="My Label" inlineHelpText="ex: (415) 111-2222" />
+		</section>
+	))
+	.add('Field Level Help', () => (
+		<section>
+			<h1 className="slds-text-title_caps slds-p-vertical--medium">
+				Field Level Help
+			</h1>
+			<Input
+				label="My Label"
+				fieldLevelHelpTooltip={
+					<Tooltip
+						align="top left"
+						content="Some helpful information"
+						position="overflowBoundaryElement"
```

Comment:
I have it so that a default trigger is used if none is specified here because it's usually going to be the "i" info icon.  What do you think?

Also, I saw that `slds-form-element__icon` adds `position: relative` to the trigger, which made the popover width much smaller.  Using `overflowBoundaryElement` sizes the width to the tooltip contents.